src/search_api/index.jsx

```diff
@@ -8,15 +8,20 @@ export class APICityModel {
   constructor(apiCityEntry) {
     this.city = apiCityEntry.City;
     this.zipcode = apiCityEntry.Zipcode;
-    // TODO: Add all required fields
+    // added required fields
+    this.state = apiCityEntry.state;
+    this.latitude = apiCityEntry.latitude;
+    this.longitude = apiCityEntry.longitude;
+    this.population = apiCityEntry.population;
+    this.totalWages = apiCityEntry.totalWages;
```

Comment:
⚠️ These `apiCityEntry` don't have those attributes ⚠️ 
The concept is correct but the syntax is wrong. If you double check the issue #1 @shikdernyc created, he included the syntax for each object attributes. It is also case-sensitive! 
For example:
We want our object to have the latitude property but from the api that we are hitting, they call their latitude property `Lat`. So we would enter `this.latitude = apiCityEntry.Lat`.

Reply:
hey @xuejingao thanks for the review!
I have updated the syntax correctly now. please check. I have also fixed the typo. thanks, for pointing that out!